BuildResidentialHPXML/measure.rb

```diff
@@ -3437,7 +3419,7 @@ def self.set_building_construction(hpxml, runner, args)
       number_of_conditioned_floors_above_grade = 1
       number_of_conditioned_floors = 1
     else
-      number_of_conditioned_floors_above_grade = args[:geometry_num_floors_above_grade]
+      number_of_conditioned_floors_above_grade = args[:geometry_unit_num_floors_above_grade]
```

Comment:
Since we're not assigning a value of 1 to `args[:geometry_unit_num_floors_above_grade]` for a MF unit, the argument might have a value other than 1 and be used incorrectly elsewhere in the measure. For example, I see at least one reference to this argument in the eaves method. Should we assign it to 1 here instead? E.g.:

```
if args[:geometry_unit_type] == HPXML::ResidentialTypeApartment
  args[:geometry_unit_num_floors_above_grade] = 1
end
number_of_conditioned_floors_above_grade = args[:geometry_unit_num_floors_above_grade]
number_of_conditioned_floors = number_of_conditioned_floors_above_grade
if args[:geometry_foundation_type] == HPXML::FoundationTypeBasementConditioned
  number_of_conditioned_floors += 1
end
```